sources/AppBundle/Controller/Website/HtmlSitemapController.php

```diff
@@ -48,7 +48,7 @@ private function buildLeafs(Branche $branche, int $id): array
 
         $pages = [];
         foreach ($leafs as $leaf) {
-            if (!$leaf['lien'] || starts_with($leaf['lien'], 'http')) {
+            if (!$leaf['lien'] || 0 === strpos($leaf['lien'], 'http')) {
```

Comment:
On a un polyfill pour la fonction `str_starts_with` :

```suggestion
            if (!$leaf['lien'] || str_starts_with($leaf['lien'], 'http')) {
```